### PR TITLE
CareKit CocoaPods support

### PR DESCRIPTION
**Pull request by rajitss**
Podspec file + changes needed to build OCKTest and CareKit for the simulator via CocoaPods.

This was requested via #1 .

### CODE REVIEW

File: CareKit.podspec [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  s.author                = { 'carekit.org' => 'http://carekit.org' }
+  s.platform              = :ios, '9.0'
+  s.ios.deployment_target = '9.0'
+  s.source                = { :git => 'https://github.com/carekit-apple/carekit.git', :tag => s.version.to_s }
+  s.source_files          = 'CareKit/**/*.{h,m}'
+  s.private_header_files  = [ 'CareKit/**/*Private.h', 'CareKit/**/*Internal.h' ]
+  s.resources             = [ 'CareKit/Assets.xcassets', 
+                              'CareKit/Localization/*.lproj', 
+                              'CareKit/CarePlan/OCKCarePlanStore.xcdatamodeld' ]
+  s.exclude_files         = [ 'docs', 'Sample', 'DerivedData' ]
```
Comment (YuanZhu-apple) [inline CareKit.podspec]: `testing` folder should not be included too.

File: CareKit.podspec [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  s.screenshots           = [ 'http://carekit.org/docs/docs/Overview/OverviewImages/CareCard.png',
+                              'http://carekit.org/docs/docs/Overview/OverviewImages/Evaluations.png',
+                              'http://carekit.org/docs/docs/Overview/OverviewImages/Dashboard.png' ]
+  s.license               = { :type => 'BSD', :file => 'LICENSE' }
+  s.author                = { 'carekit.org' => 'http://carekit.org' }
+  s.platform              = :ios, '9.0'
+  s.ios.deployment_target = '9.0'
+  s.source                = { :git => 'https://github.com/carekit-apple/carekit.git', :tag => s.version.to_s }
+  s.source_files          = 'CareKit/**/*.{h,m}'
+  s.private_header_files  = [ 'CareKit/**/*Private.h', 'CareKit/**/*Internal.h' ]
```
Comment (YuanZhu-apple) [inline CareKit.podspec]: Internal headers should not be exposed as private headers.

Comment (YuanZhu-apple) [inline CareKit.podspec]: Fine with current solution to make both of them private to Pod user.

File: CareKit.podspec (hunk 1 above)
Comment (rajitss) [inline CareKit.podspec]: Ok. Just pushed a change for that.

**Comment by YuanZhu-apple** [comment]
r=me. I will get it pushed to CocoaPod later.

**Pull request merged**